roles/edge-image-builder/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Create eib images directory
+  become: true
```

Comment:
Is become really needed?

Reply:
yesterday I started without using 'become', but after testing some parts I was starting having issues. I will re-test it again to see if we can remove some of them.